torch/csrc/jit/tracer.h

```diff
@@ -71,6 +71,10 @@ inline Value* getValueTrace(const Variable& var) {
     constant->inferTypeFrom(var.data());
     it = value_map.emplace_hint(it, var, constant);
   }
+  auto unique_name = getTracingState()->lookup_var_name_fn(var);
```

Comment:
Can we memoize it?

Reply:
How about we just skip all of this if the Value already has a unique name?

Reply:
Oh that's a perfect way to memoize them!